assets/js/works.js

```diff
@@ -0,0 +1,87 @@
+const works = [
+  {
+    heading: 'Data Dashboard Healthcare',
+    desc: "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard",
+    languages: [
+      'html',
+      'bootstrap',
+      'ruby',
+    ],
+  },
+  {
+    heading: 'Website Portfolio',
+    desc: "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard",
+    languages: [
+      'html',
+      'bootstrap',
+      'ruby',
+    ],
+  },
+  {
+    heading: 'Professional Art Printing Data More',
+    desc: "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard",
+    languages: [
+      'html',
+      'bootstrap',
+      'ruby',
+    ],
+  },
+];
+
+function displayCards() {
+  works.forEach((work, index) => {
+    let langs = '';
+    work.languages.forEach((lang) => {
+      langs += `<li>${lang}</li>`;
+    });
+
+    document.querySelector('.works').innerHTML += `
+    <div class="work-card work-card-${index}">
+      <div class="work-card-heading">
+        <h4>${work.heading}</h4>
+      </div>
+      <div class="work-card-desc">
+        <p>${work.desc}</p>
+      </div>
+      <div class="work-card-tools">
+        <ul>
+          ${langs}
+        </ul>
+      </div>
+      <div class="work-card-button">
+        <button id="${index}">See Project</button>
+      </div>
+    </div>
+    `;
```

Comment:
- [ ] For the pop-up work card design in the desktop version you forgot to add the project title on the top of the card and the project description suppose to be beside the project image not below it, So Please fix that and try to stick as much as possible to the given Figma design.

| Figma Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/68796232/168064222-28a06ae6-c531-4d80-bcc6-d1b1d11c6066.png)  |![image](https://user-images.githubusercontent.com/68796232/168064115-19409307-8308-42dd-bda7-2325abfcdb7e.png) |

- [ ] In the mobile version of the pop-up card please fix the design to be responsive and prevent side-scrolling and try to stick as much as possible to the given Figma design.

| Figma Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/68796232/168064768-55492093-38b8-40e5-8050-425cdb656828.png)  | ![chrome-capture-2022-4-12](https://user-images.githubusercontent.com/68796232/168064917-2fd0d363-7acc-45ff-a80f-894827e05246.gif) |

Reply:
- [ ] @Team, great work implementing the changes requested, there is still some differences in comparison to the figma design check the comparison below:
Desktop


Figma Design          |  Your design     
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/61408860/168105273-3e37445a-08c0-4982-ac4c-7dffdf7acdb5.png) |  <img width="517" alt="image" src="https://user-images.githubusercontent.com/61408860/168105554-2f5054cb-124c-469b-9544-1bd9284e1e30.png">

In mobile design kindly remove the extra space below the `see Iive` and `see code` buttons.Fix the design to be responsive and prevent side-scrolling

Figma Design          |  Your design     
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/61408860/168106302-22c2cf13-730b-448d-a8b6-5c1a899a7935.png)|  <img width="200" alt="image" src="https://user-images.githubusercontent.com/61408860/168106675-a81b1f35-af73-4e7b-b787-68b3b0fd3500.png">